sample/src/main/kotlin/com/thefuntasty/donutsample/ui/playground/PlaygroundActivity.kt

```diff
@@ -0,0 +1,216 @@
+package com.thefuntasty.donutsample.ui.playground
+
+import android.os.Bundle
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/donut/blob/212ed880cccf1196397e5b12a6300bab95c4049d/sample/src/main/kotlin/com/thefuntasty/donutsample/ui/playground/PlaygroundActivity.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>